### ISSUE TITLE
Dead project?

### ISSUE THREAD

**Issue by Linkachus17** [gennariarmando/v-hud#133]
It's been 1 year since last update on this mod (Last update: March 18th 2022). I think the owner of this mod abandoned the mod at least judging from the last update and no replies to Issues tab

It still has several issues:
-Controls are not binded properly on Settings, pressing anything will bind it to LMB for some reason?
-Missing help tip sound
-Seems to have issue with Modloader? Modloader menu will simply show blank and often times it'll crash if you open In-Game
-Using Skin Loader mod is almost impossible to use due to different controls and insanely broken menu

**Comment by LazytownSpo666** [comment]
Yeah, VHud and its owner. are on indefinite hiatus, which means the owner is currently inactive and cannot reply to the issues. all releases however, are still intact and available for download.
